lxc/utils.go

```diff
@@ -239,6 +243,178 @@ func getConfig(args ...string) (map[string]string, error) {
 	return values, nil
 }
 
+// stringToTimeHookFunc is a custom decoding hook that converts string values to time.Time using the given layout.
```

Comment:
Can you move these functions into called something like utils_properties.go to help organise the utils a bit better.